app/models/concerns/scripts/import.rb

```diff
@@ -58,6 +58,8 @@ def import_script data, scripts_data
           script = create_from_data data
         end
 
+        script.imported_version = data['version']
```

Comment:
Acá le pondría algo como:
```suggestion
        script.imported_version = data['version'] || MonitorApp::Application::VERSION
```
Así sabemos que siempre le mandamos algún valor cuando importamos.

Reply:
Genial. Como dijimos en el mail y charla cuando viene sin nada le asigno el '1.0.0'. Lo saque a un metodo que sea default_version, nose que te parece?

Reply:
Buenísimo, si, me pinta como quedó así con los métodos y ese delete.